operations/iuf/workflows/managed_rollout.md

```diff
@@ -59,15 +59,30 @@ referred to as the "canary node" in the remainder of this section. Use `--limit-
 
 1. Verify the canary node booted successfully with the desired image and CFS configuration.
 
-1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on all nodes, rebooting the nodes in the default staged manner in conjunction with the workload manager. If an
-immediate reboot of compute nodes is desired instead, add `-mrs reboot` to the `iuf run` command.
+1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on all nodes. This will stage data to BOS and allow the workload manager to reboot the nodes when it is ready to do so. The workload managers\
+must be configured to tell BOS to reboot nodes using this staged data. See ['Using staged sessions with Slurm'](<../../boot_orchestration/Rolling_Upgrades.md#staged-sessions-with-slurm>) for this configuration.  This assumes Slurm as the workload manager.\
```

Comment:
Why the `<` and `>` surrounding the link target? It looks like it still works, but I haven't seen this used before, and I don't think it's correct.